brand-design-guide.md

```diff
@@ -0,0 +1,76 @@
+- There is not enough information about how to use brand elements in ways that follow accessibility guidelines.
+
+### Audiences and use cases:
+
+- Designers who are working on the brand itself (maintaining source files, creating templates, updating guidance, etc.)
+- Designers who are working with the brand, working with internal stakeholders to create unique expressions of the brand for various marketing and communications purposes.
+- Non-designers who are working with the brand, primarily using templates and other provided collateral for various marketing and communications purposes that do not require a designer to be directly involved.
+
+## Solution
+
+A publicly available website that will organize information about the brand in a clear, accessible, and useful way so various stakeholders can find what they need and get guidance on how to use it.
+
+The brand guide is split up into the following sections:
+
+- logo
+- color
+- typography
+- imagery
+- templates
+
+All of the sections were created with updated guidance and information, as well as new graphics that demonstrate and help visualize how to use the brand.
+
+### New additions to the brand
+
+The PDF version of the brand had some outdated and sometimes confusing information. To remedy this, the team created new assets and content that better supports the brand.
+
+- New guidance showing [how to use the colors](https://bixal.github.io/brand-guide/color/) in ways that align with accessibility best practices was added in.
+
+- New guidance on how to use Bixal's 'B' monogram was created as a [template for social posts](https://www.figma.com/file/uSdGkDjy2OVLxlhp1NVzM4/Social-Media-Template?node-id=54%3A190). The template was made in Figma and includes a bleed guide with instructions, examples of various social posts, and downloadable frames that people can edit and post.
+
+- A [Bixal branded presentation](https://bixal365.sharepoint.com/:p:/g/EfPTLplrHSZAp8hMvkvS-y4BqfSoUka2Ro9kNV_vMXn_XA) came highly suggested from others at Bixal. So our guide includes presentation templates in both Powerpoint and [Keynote](https://bixal365.sharepoint.com/Shared%20Documents/2020%20Rebrand%20Templates/Presentation%20Templates/Bixal_Keynote_Template_2021.key).
+
+- An [Adobe asset library](https://assets.adobe.com/public/0b60fc60-a474-4b51-40e8-1d5e1b0af038) was created. Now, any Bixalers who work in the Adobe Creative Suite can have access to the various brand assets within whatever programs they use.
+
+## Results
+
+[View the new brand guide](https://bixal.github.io/brand-guide/){: .usa-button}
+
+If you have any questions or feedback, feel free to [contribute to the discussion](https://github.com/Bixal/brand-guide/issues) in Github by creating a new issue.
+
+## Tools
+
+This site is currently hosted on Github. We used Jekyll, HTML, the United States Web Design System, Bixal CSS, and markdown to bring it to life. Our team also used Figma to ideate and Microsoft Teams to collaborate.
```

Comment:
Just me being nit-picky: Having USWDS all spelled out feels a little awkward to me 😂  I'm used to calling it "U.S. Web Design System" but I'm open to keeping it all spelled out. Thoughts?